src/credentialStore.ts

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+"use strict";
+import * as keytarType from "keytar";

```

Comment:
keytar is not added as dependency right?

Reply:
Yes, keytar is shipped with VS Code.

Reply:
From getCoreNodeModule, it seems keytar is not in normal nodemodule, So shall we import here? From sample in https://code.visualstudio.com/api/advanced-topics/remote-extensions, it doesn't import keytar

Reply:
Importing keytarType is just for type (for intellisense), refer to https://github.com/Microsoft/vscode-azure-ext-sdk#access-os-credential-stores